EIPS/eip-5334.md

```diff
@@ -0,0 +1,197 @@
+---
+eip: 5334
+title: 721 User And Expires And Level Extension
```

Comment:
```suggestion
title: ERC-721 User And Expires And Level Extension
```
Although I think this title can be improved - it isn't quite clear to me what is being proposed.

Reply:
![image](https://user-images.githubusercontent.com/37472467/181148112-d2041b1e-2ad3-4a65-95f5-62c007ee3824.png)